datasets/swedish_medical_ner/README.md

```diff
@@ -0,0 +1,192 @@
+```
+
+### Data Fields
+
+- `sentence`
+- `entities`
+    - `start`: the start index
+    - `end`: the end index
+    - `text`: the text of the entity
+    - `type`: entity type (Disorder and Finding / Pharmaceutical Drug / Body Structure)
+
+### Data Splits
+
+In the original paper, its authors used the text from Läkartidningen for model training, Swedish Wikipedia for validation, and 1177.se for the final model evaluation. 
+
+## Dataset Creation
+
+### Curation Rationale
+
+### Source Data
+
+- Swedish Wikipedia;
+- Läkartidningen - contains articles from the Swedish journal for medical professionals;
+- 1177.se - a web site provided by the Swedish public health care authorities, containing information, counselling, and other health-care services.
+
+#### Initial Data Collection and Normalization
+
+[More Information Needed]
+
+#### Who are the source language producers?
+
+[More Information Needed]
+
+### Annotations
+
+#### Annotation process
+
+- A list of seed terms was extracted using SweMeSH and SNOMED CT;
+   - The following predefined categories was used for the extraction: disorder & finding (sjukdom & symtom), pharmaceutical drug (läkemedel) and body structure (kroppsdel)
+- For _Swedish Wikipedia_, an initial list of medical domain articles were selected manually. These source articles as well as their linked articles were downloaded and automatically annotated by finding the aforementioned seed terms with a context window of 60 characters;
+- Articles from the _Läkartidningen_ corpus were downloaded and automatically annotated by finding the aforementioned seed terms with a context window of 60 characters;
+- 15 documents from _1177.se_ were downloaded in May 2016 and then manually annotated with the seed terms as support, resulting 2740 annotations.
+
+#### Who are the annotators?
+
+[More Information Needed]
+
+### Personal and Sensitive Information
+
+[More Information Needed]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+[More Information Needed]
+
+### Discussion of Biases
+
+[More Information Needed]
+
+### Other Known Limitations
+
+[More Information Needed]
+
+## Additional Information
+
+### Dataset Curators
+
+- Simon Almgren, simonwalmgren@gmail.com
+- Sean Pavlov, sean.pavlov@gmail.com
+- Olof Mogren, olof@mogren.one
+Chalmers University of Technology
+
+### Licensing Information
+
+This dataset is released under the [Creative Commons Attribution-ShareAlike 4.0 International Public License (CC BY-SA 4.0)](http://creativecommons.org/licenses/by-sa/4.0/).
+
+### Citation Information
+@inproceedings{almgrenpavlovmogren2016bioner,
+  title={Named Entity Recognition in Swedish Medical Journals with Deep Bidirectional Character-Based LSTMs},
+  author={Simon Almgren, Sean Pavlov, Olof Mogren},
+  booktitle={Proceedings of the Fifth Workshop on Building and Evaluating Resources for Biomedical Text Mining (BioTxtM 2016)},
+  pages={1},
+  year={2016}
+}
```

Comment:
````suggestion
```bibtex
@inproceedings{almgrenpavlovmogren2016bioner,
  title={Named Entity Recognition in Swedish Medical Journals with Deep Bidirectional Character-Based LSTMs},
  author={Simon Almgren, Sean Pavlov, Olof Mogren},
  booktitle={Proceedings of the Fifth Workshop on Building and Evaluating Resources for Biomedical Text Mining (BioTxtM 2016)},
  pages={1},
  year={2016}
}
```
````